### ISSUE TITLE
Cascading World Gen on 1.6.0

### ISSUE THREAD

**Issue by TLomon** [NFINIT-Development/Astroblock2#31]
Installed v1.6.0 today and started a new world (actually a few to verify this is happening and wasn't a one off)

Snippet:
07:51:36.899
Recurrent Complex loaded a new chunk [-8, -13] in dimension -30 (planet.asteroids) while populating chunk [-7, -13], causing cascading worldgen lag.
07:51:36.899
Please report this to the mod's issue tracker. This log can be disabled in the Forge config.
07:51:40.408
Recurrent Complex loaded a new chunk [-10, 9] in dimension -30 (planet.asteroids) while populating chunk [-9, 9], causing cascading worldgen lag.
07:51:40.409
Please report this to the mod's issue tracker. This log can be disabled in the Forge config.
07:51:48.406
Recurrent Complex loaded a new chunk [-8, 12] in dimension -30 (planet.asteroids) while populating chunk [-8, 11], causing cascading worldgen lag.
07:51:48.407
Please report this to the mod's issue tracker. This log can be disabled in the Forge config.
07:51:54.038
Can't keep up! Did the system time change, or is the server overloaded? Running 26512ms behind, skipping 530 tick(s)
07:51:54.070
Recurrent Complex loaded a new chunk [11, 11] in dimension -30 (planet.asteroids) while populating chunk [11, 10], causing cascading worldgen lag.

Basically, the world loads fine. Then the new military ships start spawning... and spawning... and spawning. I counted twelve surrounding the starting ship before my system started getting too slow. Was a combination of damaged and new (mostly damaged). By surrounded, I mean they are right there. In all directions.

As stated, I created new worlds to verify this. It duplicated each time.

**Comment by TLomon** [comment]
Fixed in 1.6.1. (Please let me know if you would rather close tickets yourself.)
